2by2.py

```diff
@@ -49,4 +67,4 @@ def turn_u(self):
 print(cube.face_to_str('l'))
 print(cube.face_to_str('r'))
 print(cube.face_to_str('f'))
-print(cube.face_to_str('b'))
\ No newline at end of file
+print(cube.face_to_str('b'))
```

Comment:
Please resolve the conflicts that would occur when merging, as you have a super outdated copy of master in your branch.